accounts/abi/method.go

```diff
@@ -32,7 +32,19 @@ import (
 // be flagged `false`.
 // Input specifies the required input parameters for this gives method.
 type Method struct {
-	Name    string
+	// Name the name of the method used for internal representation. which
+	// It's derived from raw name and will be added suffix when function
+	// overload occurs.
```

Comment:
Name is **the method** name used for internal representation. *looks like there is an extra, dangling "which" here*
It's derived from **the** raw name and **a suffix will be added in the case of a function overload**